simson/common/data_extrapolations.py

```diff
@@ -79,6 +82,8 @@ def regress(self):
         target_shape = tuple([self.target_range.shape[i] for i in sorted(self.independent_dims)])
         regression = np.zeros_like(self.target_range)
         self.fit_prms = np.zeros(self.target_range.shape[1:] + (self.n_prms,))
+        bound_shape = tuple(self.target_range.shape[i] for i in self.independent_dims)
```

Comment:
Is this necessarily the shape that the bounds have? 
Should that be ensured/checked somewhere?